pkgs/tools/security/onioncircuits/default.nix

```diff
@@ -1,31 +1,54 @@
-{ stdenv, lib, fetchgit, python3, intltool, gtk3, gobject-introspection, gnome }:
+{ stdenv
+, lib
+, fetchgit
+, gnome
+, gobject-introspection
+, gtk3
+, intltool
+, python3
+, wrapGAppsHook
+}:
 
 python3.pkgs.buildPythonApplication rec {
   pname = "onioncircuits";
-  version = "0.5";
+  version = "0.7";
 
   src = fetchgit {
-    url = "https://git-tails.immerda.ch/onioncircuits/";
+    url = "https://gitlab.tails.boum.org/tails/onioncircuits";
     rev = version;
     sha256 = "13mqif9b9iajpkrl9ijspdnvy82kxhprxd5mw3njk68rcn4z2pcm";
   };
 
-  nativeBuildInputs = [ intltool ];
-  buildInputs = [ gtk3 gobject-introspection ];
-  propagatedBuildInputs =  with python3.pkgs; [ stem distutils_extra pygobject3 ];
+  format = "other";
```

Comment:
Looks like a setuptools build to me, which would make overriding the phases unnecessary.

Reply:
It's a legacy distutils build, but it uses a source layout that setuptools is unhappy with, which is the current source of trouble with the build. Overriding the phases lets us use distutils directly (until Python drops it in 3.12).

Reply:
The problem was pointed out on the upstream too. https://gitlab.tails.boum.org/tails/onioncircuits/-/merge_requests/4
Fixed with the patch.